tests/test_hdfs.py

```diff
@@ -324,20 +111,20 @@ def wait_job_present(present):
 
 @pytest.mark.skipif(not utils.hdfs_enabled(), reason='HDFS_ENABLED is false')
 @pytest.mark.sanity
-def test_history():
+def test_history(kerberized_spark, hdfs_with_kerberos, setup_history_server):
```

Comment:
if you're depending on`kerberized_spark` (which internally depends on `setup_history_sever, hdfs_with_kerberos`), do you need to call those two again here?

Reply:
okay that's fair